src/CameraControls.ts

```diff
@@ -2111,28 +2112,16 @@ export class CameraControls extends EventDispatcher {
 	}
 
 	/**
-	 * Similar to setLookAt, but it interpolates between two states.
-	 * @param positionAX
-	 * @param positionAY
-	 * @param positionAZ
-	 * @param targetAX
-	 * @param targetAY
-	 * @param targetAZ
-	 * @param positionBX
-	 * @param positionBY
-	 * @param positionBZ
-	 * @param targetBX
-	 * @param targetBY
-	 * @param targetBZ
+	 * Interpolates between two states.
+	 * @param stateA
+	 * @param stateB
 	 * @param t
 	 * @param enableTransition
 	 * @category Methods
 	 */
-	lerpLookAt(
-		positionAX: number, positionAY: number, positionAZ: number,
-		targetAX: number, targetAY: number, targetAZ: number,
-		positionBX: number, positionBY: number, positionBZ: number,
-		targetBX: number, targetBY: number, targetBZ: number,
+	lerp(
+		stateA: CameraControlsLerpState,
+		stateB: CameraControlsLerpState,
```

Comment:
LGTM for the args.
Previously I preferred the flattened format for required args, but it felt a bit messy...😵‍💫